src/main/java/ResultView.java

```diff
@@ -1,21 +1,37 @@
-class ResultView {
-    private static String textOfResult = "결과보기";
-    private RacingCarCompetition competition;
+import java.util.List;
+import java.util.stream.Collectors;
 
-    ResultView(RacingCarCompetition competition) {
+public class ResultView {
+    private final static String textOfResult = "결과보기";
+    private final static String suffixOfText = "가 최종 우승했습니다.";
+
+    public List<List<Car>> histories;
+
+    public ResultView(List<List<Car>> histories){
+        this.histories = histories;
+    }
+
+    public void getHistories() {
+        for (List<Car> car: histories) {
+            showPositions(car);
+        }
+    }
+
+    public void showPositions(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.println(car.getName() + ": " + car.getPositionString());
+        }
         System.out.println();
+    }
+
+    public void showResultsMessage() {
         System.out.println(textOfResult);
-        this.competition = competition;
     }
 
-    public void showDistance() {
-        int historySize = competition.history.size();
+    public void showWinner(List<Car> winners) {
+        String alertOfWinner = winners.stream().map(Car::getName).collect(Collectors.joining(", "));
```

Comment:
스트림 이용 잘해주셨어요!👍
```suggestion
        String alertOfWinner = winners.stream()
                .map(Car::getName)
                .collect(Collectors.joining(", "));
```
요렇게 점을`.` 기준으로 줄바꿈하는 방식이 일반적이랍니다😁